src/web/wms/src/main/resources/GeoServerApplication_ja.properties

```diff
@@ -53,6 +53,7 @@ WMSAdminPage.bboxForEachCRSHelp.title=CRS\u3054\u3068\u306e\u7bc4\u56f2\u77e9\u5
 WMSAdminPage.bboxForEachCRSHelp.message=\u305d\u308c\u305e\u308c\u306eCRS\u30d5\u30e9\u30b0\u306b\u5bfe\u3059\u308b\u77e9\u5f62\u7bc4\u56f2\u306f\u9650\u5b9aSRS\u30ea\u30b9\u30c8\u304c\u6307\u5b9a\u3055\u308c\u305f\u5834\u5408\u306e\u307f\u8a55\u4fa1\u3055\u308c\u307e\u3059\u3002\u30d5\u30e9\u30b0\u3092\u8a2d\u5b9a\u3059\u308b\u3053\u3068\u306b\u3088\u308a\u3001WMS\u30ad\u30e3\u30d1\u30d3\u30ea\u30c6\u30a3\u30c9\u30ad\u30e5\u30e1\u30f3\u30c8\u306f\u5bfe\u5fdc\u3059\u308b\u305d\u308c\u305e\u308c\u306eCRS\u304a\u3088\u3073\u305d\u308c\u305e\u308c\u306e\u30ec\u30a4\u30e4\u30fc\u306b\u5bfe\u3057\u3066\u77e9\u5f62\u7bc4\u56f2\u3092\u542b\u3081\u307e\u3059\u3002EPSG\u30c7\u30fc\u30bf\u30d9\u30fc\u30b9\u306b\u8a18\u8f09\u3055\u308c\u3066\u3044\u308b\u3059\u3079\u3066\u306eCRS\u3001\u304a\u3088\u3073\u30ab\u30bf\u30ed\u30b0\u5185\u306e\u305d\u308c\u305e\u308c\u306e\u30ec\u30a4\u30e4\u30fc\u306b\u3053\u306e\u4f5c\u696d\u3092\u884c\u3046\u3068\u3001\u30ad\u30e3\u30d1\u30d3\u30ea\u30c6\u30a3\u30c9\u30ad\u30e5\u30e1\u30f3\u30c8\u306f\u975e\u5b9f\u7528\u7684\u306a\u307b\u3069\u5de8\u5927\u306b\u306a\u308a\u307e\u3059\u3002
 WMSAdminPage.defaultInterpolation   = \u898f\u5b9a\u306e\u88dc\u5b8c
 WMSAdminPage.description            = \u5730\u56f3\u306e\u516c\u958b\u7ba1\u7406
+WMSAdminPage.disposalMethod         = Disposal Method
```

Comment:
Please leave the strings in the other languages as missing, the code should automatically display the english version if the string was not found